Sources/PrimerSDK/Classes/PCI/Checkout Components/PrimerCardData.swift

```diff
@@ -32,20 +32,32 @@ public class PrimerCardData: PrimerRawData {
         }
     }
     
+    public var cardNetworkIdentifier: String? {
+        didSet {
+            self.onDataDidChange?()
+        }
+    }
+    
     private enum CodingKeys: String, CodingKey {
-        case cardNumber, expiryDate, cvv, cardholderName
+        case cardNumber,
+             expiryDate,
+             cvv,
+             cardholderName,
+             cardNetworkIdentifier
     }
         
     public required init(
         cardNumber: String,
         expiryDate: String,
         cvv: String,
-        cardholderName: String?
+        cardholderName: String?,
+        cardNetworkIdentifier: String?
```

Comment:
Since `PrimerCardData` is public, it might be wise to give `cardNetworkIdentifier` a default value of nil, so as not to introduce a breaking change.

Reply:
Ah good catch, this was the intention but it slipped by me. Will change 👍